Java/sort/SORT_MATRIX.java

```diff
@@ -0,0 +1,80 @@
+package com.company;

```

Comment:
Remove Package

Reply:
Not yet resolved.

Reply:
I have added the new code. Can you check?